image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -265,11 +265,11 @@ defaults:
                 operator: DoesNotExist
       resources:
         requests:
-          memory: "500Mi"
-          cpu: "1000m"
+          memory: "1500Mi"
+          cpu: "500m"
```

Comment:
I think this is ok, as we really care more about the DB's resources, as updating and matching is very DB heavy. The matcher really doesn't do too much itself

Reply:
> The matcher really doesn't do too much itself
These new requests are more aligned with what was observed in the image scanning tests, so ensuring these resources are available makes sense to me.